src/string.rs

```diff
@@ -121,7 +121,7 @@ extern "C" fn one_byte_const_data(this: *const OneByteConst) -> *const char {
 }
 extern "C" fn one_byte_const_length(this: *const OneByteConst) -> usize {
```

Comment:
Just to sanity check -- should this return `int`? I checked the v8 ref and it's a 4-byte return value. While this _shouldn't_ matter in most cases, should we make this int as well?